release_notes/ocp-4-16-release-notes.adoc

```diff
@@ -234,6 +234,10 @@ cgroup v1 is still used in upgraded clusters with performance profiles that have
 
 For more information, see xref:../nodes/clusters/nodes-cluster-cgroups-2.adoc#nodes-clusters-cgroups-2[Configuring the Linux cgroup version on your nodes].
 
+[id="ocp-4-16-etcd-increase-db-size"]
```

Comment:
Courtney recently went through to add contexts to these, would be good to add here too!

```suggestion
[id="ocp-4-16-etcd-increase-db-size_{context}"]
```

Reply:
Done!